src/docs/output-targets/custom-elements.md

```diff
@@ -6,51 +6,44 @@ contributors:
   - adamdbradley
   - rwaskiewicz
   - splitinfinities
+  - tanner-reits
 ---
 
 # Custom Elements
 
-The `dist-custom-elements` output target is used to generate custom elements in a more optimized way for tree shaking, and its the recommended approach when using any frontend framework integrations. For example, if a component library has 100 components, but an external project only imported one component from the directory, then only the code used by that one component would be pulled into the project. This is due to Stencil's use of ES Modules and the compiler generating friendly code for bundlers to parse and understand.
+The `dist-custom-elements` output target is used to generate custom elements in a more optimized way for tree shaking, and it's the recommended approach when using any frontend framework integrations.
```

Comment:
The "in a more optimized way for tree shaking" was originally written (albeit unclearly) to differentiate it from `dist-custom-elements-bundle`. Since we'll be removing that in the near future, I wonder if we should doctor this sentence a little bit to either
- compare it against `www`
- talk about it a little more generally - e.g. "what's it do?"

Reply:
I think we kind of address comparing it to `www` with the other callout you had for lazy-loading vs. non lazy-loading (since I feel like that's the big difference most users would make a decision against). 

What else would we generally talk about? Elaborate more on treeshaking? Talk about using Rollup under-the-hood and what that generates? In that case, we might just keep what was originally there but wordsmith it a bit

Reply:
I think that currently (and before this PR) we didn't really do a good job at giving a high level description of this output target. It doesn't feel we didn't really answer "What is this?", rather we compared it to `dist-custom-elements` and said it was recommended for frontend integrations. I don't think I'm describing it well, but it feels like there's an executive summary/elevator pitch for this output target that could be added here. Does that make sense?

Reply:
Alright, added a few sentences that I morphed from a few other doc pages that were doing some of these comparisons